src/pages/library/dataBrowser-utils.js

```diff
@@ -80,7 +80,7 @@ const normalizeDataset = dataset => {
     dataReleasePolicy,
     contacts, curators, contributorNames,
     dataType, dataModality,
-    access: _.intersection(dataset.roles, ['reader', 'owner']).length > 0 ? datasetAccessTypes.GRANTED : datasetAccessTypes.CONTROLLED
+    access: _.intersection(_.map(role => _.toLower(role), dataset.roles), ['reader', 'writer', 'owner']).length > 0 ? datasetAccessTypes.GRANTED : datasetAccessTypes.CONTROLLED
```

Comment:
ESLint is complaining about the unnecessary wrapper function.

```suggestion
    access: _.intersection(_.map(_.toLower, dataset.roles), ['reader', 'writer', 'owner']).length > 0 ? datasetAccessTypes.GRANTED : datasetAccessTypes.CONTROLLED
```